torch/_inductor/lowering.py

```diff
@@ -4231,14 +4306,16 @@ def fn_sum(idx, loader):
         if divisor_override:
             scale = 1 / divisor_override
         else:
-            scale = 1.0 / (kernel_size[0] * kernel_size[1])
+            scale = 1.0 / window_size
 
         def fn(idx):
             return ops.mul(fn_sum(idx, x_loader), ops.constant(scale, dtype))
 
     else:
-        ones_loader = constant_boundary_condition_2d(
-            ones_like(x), 0.0, padding if count_include_pad else None
+        ones_loader = constant_boundary_condition(
+            ones_like(x),
+            1.0 if count_include_pad else 0.0,
+            dim=dim,
```

Comment:
Test failures are real and this is the most suspicious looking change, are you sure this is correct?